src/main/java/book/store/config/SecurityConfig.java

```diff
@@ -0,0 +1,39 @@
+package book.store.config;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@EnableMethodSecurity
+@RequiredArgsConstructor
+@Configuration
+public class SecurityConfig {
+    private final UserDetailsService userDetailsService;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        return http
+            .cors(AbstractHttpConfigurer::disable)
+            .csrf(AbstractHttpConfigurer::disable)
+            .authorizeHttpRequests(
+                auth -> auth.requestMatchers("/auth/**", "/error", "/swagger-ui/**")
+                .permitAll().anyRequest().authenticated()
```

Comment:
Please try to format code so it has new line between logical compositions, something like
```suggestion
            .authorizeHttpRequests(auth -> auth
                .requestMatchers("/auth/**", "/error", "/swagger-ui/**").permitAll()
                .anyRequest().authenticated()
```